Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -2970,59 +2970,31 @@ - (void)yapDatabaseModified:(NSNotification *)notification
 
     DDLogVerbose(@"self.viewItems.count: %zd -> %zd", oldViewItemCount, self.viewItems.count);
 
-    BOOL shouldReloadCollection = [self shouldReloadCollection:rowChanges];
```

Comment:
We don't need this hack either now, I believe.  Moves are working properly now that we're reloading with the original indices.